db/primary_migrate/20240401185713_add_check_password_compromised_at_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddCheckPasswordCompromisedAtToUsers < ActiveRecord::Migration[7.1]
+  def change
+    add_column :users, :check_password_compromised_at, :datetime, default: nil
```

Comment:
Minor: We're not super consistent with naming, but I'd typically expect datetime fields formatted as `[thing]_[verb]ed_at`, with the primary verb coming right before "at". Since we're more interested in this reflecting the check and not the date at which the password became compromised, I'd wonder if this should be rearranged a little:

```suggestion
    add_column :users, :compromised_password_checked_at, :datetime, default: nil
```